tools/UnitTestApp/UnitTests/LineChartUT.test.tsx

```diff
@@ -0,0 +1,534 @@
+import { fireEvent } from '@testing-library/react';
+import { ILineChartPoints, LineChart } from '../LineChart/index';
+import { DefaultPalette } from '@fluentui/react';
+import { IChartProps, ILineChartGap } from '../../index';
+import { LineChartBase } from '../LineChart/LineChart.base';
+import { getById, testWithWait, testWithoutWait } from '../../utilities/TestUtility.test';
+import { DarkTheme } from '@fluentui/theme-samples';
+import * as fs from 'fs';
+import { updatedFile } from './TestEnvSetupScript';
+
+updatedFile(fs.realpathSync('./') + '/src/components/LineChart/LineChart.base.tsx');
```

Comment:
Run this script as part of workflow once for all files under src folder.

Reply:
Running for all files will not be required, as only the base class is needed. Running for all will take up too much time as well. As a result, I am running it per test file so that we do not take up too much execution time.

Reply:
sure, will do that